infoschema/perfschema/tables.go

```diff
@@ -202,19 +189,7 @@ func (vt *perfSchemaTable) Type() table.Type {
 }
 
 func (vt *perfSchemaTable) getRows(ctx sessionctx.Context, cols []*table.Column) (fullRows [][]types.Datum, err error) {
-	// Extract user and privilege info (is super user?) here
-	// for statement summary tables' access privilege check
-	user := ctx.GetSessionVars().User
-	isSuper := false
-	if pm := privilege.GetPrivilegeManager(ctx); pm != nil {
-		isSuper = pm.RequestVerificationWithUser("", "", "", mysql.SuperPriv, user)
-	}
-
 	switch vt.meta.Name.O {
-	case tableNameEventsStatementsSummaryByDigest:
-		fullRows = stmtsummary.StmtSummaryByDigestMap.ToCurrentDatum(user, isSuper)
-	case tableNameEventsStatementsSummaryByDigestHistory:
-		fullRows = stmtsummary.StmtSummaryByDigestMap.ToHistoryDatum(user, isSuper)
 	case tableNameTiDBProfileCPU:
```

Comment:
Do we need to check privilege before use access this table?

Reply:
Privilege management on statement summary tables is very special. All users can see statements executed by themselves, and superusers can see all statements. So we can only check privilege in `ToCurrentDatum` and `ToHistoryDatum`.

Reply:
This table has nothing to do with this PR.